x/slashing/client/cli/cli_test.go

```diff
@@ -186,6 +186,50 @@ func (s *IntegrationTestSuite) TestNewActivateTxCmd() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestTxProposalResetWholeValidatorRankCmd() {
```

Comment:
Messages:   	{"height":"0","txhash":"176D994FF67A958F0F00B7311B0EFC8AED2B5519D97F282912D2375E85BB8C9D","codespace":"sdk","code":2,"data":"","raw_log":"unable to resolve type URL /kira.slashing.MsgProposalResetWholeValidatorRank: tx parse error","logs":[],"info":"","gas_wanted":"0","gas_used":"0","tx":null,"timestamp":""}


This does not work